sass/style.scss

```diff
@@ -28,3 +28,12 @@
 .form-item-text-format-value + .ecl-form-group {
   margin-top: 0;
 }
+
+// Align Apply bottom in moderation block
+.entity-moderation-form {
+  .entity-moderation-form__item {
+    button.ecl-button {
```

Comment:
Also this is overqualified, let's leave only `.ecl-button`.

Reply:
Done